starboard/shared/starboard/media/codec_util_test.cc

```diff
@@ -263,6 +263,48 @@ TEST(CodecUtilTest, DoesNotParse1AsPcmForNonWavSubtypes) {
   EXPECT_EQ(GetAudioCodecFromString("1", "webm"), kSbMediaAudioCodecNone);
 }
 
+TEST(CodecUtilTest, ParsesIamfCodec) {
+#if SB_API_VERSION < 15
+  GTEST_SKIP() << "IAMF is unsupported on Starboard " << SB_API_VERSION;
+#endif  // SB_API_VERSION < 15
+  EXPECT_EQ(GetAudioCodecFromString("iamf", ""), kSbMediaAudioCodecIamf);
+  EXPECT_EQ(GetAudioCodecFromString("iamf.000.000.Opus", ""),
+            kSbMediaAudioCodecIamf);
+  EXPECT_EQ(GetAudioCodecFromString("iamf.000.000.mp4a.40.2", ""),
+            kSbMediaAudioCodecIamf);
+  EXPECT_EQ(GetAudioCodecFromString("iamf.000.000.fLaC", ""),
+            kSbMediaAudioCodecIamf);
+  EXPECT_EQ(GetAudioCodecFromString("iamf.000.000.ipcm", ""),
+            kSbMediaAudioCodecIamf);
+}
+
+TEST(CodecUtilTest, IsIamfMimeType) {
```

Comment:
Consider to add cases where the length of resulting string vector is not 4, like `iamf.xxx.Opus`, `iamf.1xx.2yy`.

Reply:
Done